CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Releases
 
+## Version 0.14.0
+
+Date: 2022-09-30
+
+This release focuses on three main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
```

Comment:
```suggestion
This release focuses on three main themes: support for running Panel apps entirely in the browser using WebAssembly (via Pyodide and PyScript), improvements in the app-user experience by making it easier to build responsive and performant applications, and finally improvements in the developer experience through static typing and docstrings.
```

(I'm making a guess that by "quality of life" you mean "for the user of a Panel app", and that you are trying to distinguish between improvements that are visible to an app visitor and improvements for app developers. If not, please reword.)